Source/CarthageKit/Xcode.swift

```diff
@@ -1242,6 +1309,13 @@ public func buildInDirectory(directoryURL: NSURL, withOptions options: BuildOpti
 				return BuildSchemeProducer(value: .Success(initialValue))
 					.concat(buildProgress)
 			}
+			.on(completed: {
+				if let dependency = dependency, rootDirectoryURL = rootDirectoryURL {
+					if !createVersionFileForDependency(dependency, forPlatforms: options.platforms, buildProductURLs: urls, rootDirectoryURL: rootDirectoryURL) {
+						observer.sendFailed(CarthageError.CanNotBuildVersionFile(dependency.project))
```

Comment:
This doesn't seem like it should be an error. If the version file can't be created, should the build really fail?